tests/certification/embedded/embedded.go

```diff
@@ -104,21 +109,21 @@ func WithComponentsPath(path string) Option {
 	return WithResourcesPath(path)
 }
 
-func WithProfilePort(port int) Option {
+func WithProfilePort(port string) Option {
```

Comment:
Why changing all ports from int to string?

Reply:
This is what the runtime config accepts.

Reply:
But before we were accepting ints here and then converting to string in a single place. This change required changing all cert tests to convert from string to int themselves. Not the end of the world but just seems more verbose

Reply:
It’s because we have removed the runtime Options completely, so we are using the same struct which flags are being used against in dapr/dapr

Reply:
Ok